### ISSUE TITLE
Client continuously sends data for current position

### ISSUE THREAD

**Issue by remorses** [voxelize/voxelize#31]
Here is a video of the packets sent, they contain the character position and direction

https://github.com/voxelize/voxelize/assets/31321188/21a6f2ef-d02f-4ef3-bcb1-4deafb2a3aa9

For example:
```
00000000: 0804 3261 0a15 5a52 7a31 6175 6e30 5144  ..2a..ZRz1aun0QD
00000001: 3632 5350 3542 4c46 6d76 3612 0b47 7565  62SP5BLFmv6..Gue
00000002: 7374 2030 3239 3830 1a3b 7b22 706f 7369  st 02980.;{"posi
00000003: 7469 6f6e 223a 5b30 2e35 352c 3134 2e32  tion":[0.55,14.2
00000004: 3832 3530 3030 3030 312c 302e 3535 5d2c  825000001,0.55],
00000005: 2264 6972 6563 7469 6f6e 223a 5b30 2c30  "direction":[0,0
00000006: 2c2d 315d 7d                             ,-1]}
```
In my opinion the server and client should not send packets if nothing happened in the world

**Comment by shaoruu** [comment]
This is a good catch. Will patch it!

**Comment by shaoruu** [comment]
[Fixed in 0.1.191](https://github.com/voxelize/voxelize/commit/d4eb52759f4433d325d8f0e44c1109e178fa6f57), let me know if it doesn't work as expected.
